low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -68,16 +69,37 @@ class SSPLBundleGenerate(object):
         self.boot_drvs_dta = f'{sspl_bundle_tmp}/Server_OS_boot_drives_SMART_data_{str(int(time.time()))}'
         self.enclosure_log = f"{sspl_bundle_tmp}/enclosure_logs.zip"
         self.enclosure_wwn = "NA"
-
-    def generate_sspl_support_bundle(self, parser, tmpFile):
-        file_name = "sspl_%s.tar.gz" % parser[1]
+        # Initialize Logging
+        self._logger = self.initialize_logging()
+
+    def initialize_logging(self):
+        # set Logging Handlers
+        _logger = logging.getLogger('sspl_support_bundle')
+        _logger.setLevel(logging.INFO)
```

Comment:
Can we take log level from ConfStore.
log_level = Conf.get(SSPL_CONF, "SYSTEM_INFORMATION>log_level")